python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -646,6 +649,96 @@ async def main():
     compiled_dag.teardown()
 
 
+class TestMultiChannel:
```

Comment:
I'm a bit confused by these tests. I thought MultiChannel would need multiple nodes to read the same result? But here we are only testing chain DAGs where each node is read by exactly one other node, so why do we need MultiChannels?

For example, I thought this would require a MultiChannel:
```python
with InputNode() as inp:
  dag = a.inc.bind(inp)
  dag = MultiOutputNode([a.inc.bind(dag), b.inc.bind(dag)])
```

Reply:
Added https://github.com/ray-project/ray/pull/45591/commits/a29881118f269a66a034c17d515f76662474dc12